src/test/datascience/notebook/executionService.mock.vscode.test.ts

```diff
@@ -1,218 +1,218 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
+// // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I assume you're just going to delete this file?

Reply:
Leaving for now, this is used to write tests with a mocked Jupyter.
I never did end up using it, but leaving in case we want to use again, if after 2-3 months I dont use it ill remove it.
Been too busy to write tests, we still need a lot of tests.